src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -90,6 +90,7 @@ class JesBackend extends Backend with LazyLogging {
           case Success(gcsPath) => (fqn, WdlFile(localFilePathFromCloudStoragePath(gcsPath).toString))
           case Failure(e) => (fqn, wdlValue)
         }
+      case array:WdlArray => fqn -> array.map { v => mapInputValue("bogus", v)._2 } // TODO: this is ugly.
```

Comment:
Ah it's because of the ` -> `. There's always the other non-shunned style, `array.map(v => ???` :)